cardano-node/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -1,7 +1,7 @@
-{-# LANGUAGE FlexibleContexts #-}
-{-# LANGUAGE FlexibleInstances #-}
-{-# LANGUAGE GADTs #-}
-{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE FlexibleInstances   #-}
+{-# LANGUAGE GADTs               #-}
+{-# LANGUAGE NamedFieldPuns      #-}
```

Comment:
It's not recommended: https://github.com/input-output-hk/cardano-node/blob/master/STYLE.md#language-extensions-section

Reply:
I find the recommendation that the `cardano-node` provides dubious -- as it makes _reading_ harder.

But it's the by-laws of the particular land, and so we must abide.

Reply:
I use stylish-haskell for formatting this, is their a "node configuration version" of stylish haskell?

Reply:
I found it and adopted it.

Reply:
All the changes in this module need to be reverted as they are uneccessary